bundles/org.openhab.binding.fineoffsetweatherstation/src/main/resources/OH-INF/thing/gateway.xml

```diff
@@ -0,0 +1,166 @@
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="pressure">
+		<item-type>Number:Pressure</item-type>
+		<label>Pressure</label>
+		<category>Pressure</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Pressure</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="humidity">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Humidity</label>
+		<category>Humidity</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Humidity</tag>
+		</tags>
+		<state pattern="%.0f %%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="illumination">
+		<item-type>Number:Illuminance</item-type>
+		<label>Illumination</label>
+		<category>Sun</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Light</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="uv-radiation">
+		<item-type>Number:Intensity</item-type>
+		<label>UV-Irradiation</label>
+		<category>Sun</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Light</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="uv-index">
+		<item-type>Number:Dimensionless</item-type>
+		<label>UV-Index</label>
+		<category>Sun</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Light</tag>
+		</tags>
+		<state pattern="%.0f" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="rain-rate">
+		<item-type>Number:VolumetricFlowRate</item-type>
+		<label>Rain Rate</label>
+		<category>Rain</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Rain</tag>
+		</tags>
+		<state pattern="%.2f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="pm25">
+		<item-type>Number:Density</item-type>
+		<label>PM2.5 Air Quality</label>
+		<category>AirQuality</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>AirQuality</tag>
+		</tags>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="water-leak-detection">
+		<item-type>Switch</item-type>
+		<label>Water Leak Detection</label>
+		<category>Alarm</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Water</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="lightning-counter">
+		<item-type>Number</item-type>
+		<label>Lightning Counter</label>
+		<category>Energy</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>None</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="lightning-time">
+		<item-type>DateTime</item-type>
+		<label>Lightning Time</label>
+		<category>Energy</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>Timestamp</tag>
+		</tags>
+		<state readOnly="true"/>
+	</channel-type>
+
+	<channel-type id="lightning-distance">
+		<item-type>Number:Length</item-type>
+		<label>Lightning Distance</label>
+		<category>Energy</category>
+		<tags>
+			<tag>Measurement</tag>
+			<tag>None</tag>
+		</tags>
```

Comment:
Please remove this tag as it is pointless IMHO. Also be aware that a lot of the tags just got removed and cleaned up from some of the bindings including the ipObserver binding. See here and some of the PR have comments that explain why to get an idea of what is ok or not...

https://github.com/openhab/openhab-addons/pulls?q=is%3Apr+Cleanup+semantic+tags

Reply:
@Skinah should I remove the `<tag>Measurement</tag>` as well or just the `<tag>None</tag>`?

Reply:
Both. The tags are used by the semantic model to draw the UI and could be used to help a AI assistant in the future know more about the channels. Half complete or tags that point to a ON and OFF switch and not to an actual amount, it may cause a crash or bug to occur. I have not seen a comprehensive topic on what should and should not be done with the tags so hopefully it does exist or will be done soon.